src/content/docs/en/pages/guides/marketplace/templates/videoteca-static-cache.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: How to deploy the Static Cache + Videoteca Player template
+description: Deploy an application with static cache settings using the Videoteca Player.
+meta_tags: templates, guides, Azion Marketplace, web
+namespace: docs_guides_static_cache_videoteca_player
+permalink: /documentation/products/guides/static-cache-videoteca-player/
+---
+
+import Tag from 'primevue/tag';
+
+<Tag severity="info" client:only="vue">
+Preview
+</Tag>
+
+The **Static Cache + Videoteca Player** template enhances the delivery and performance of static and dynamic content based on file extensions. It implements the Azion **Videoteca Player** integration for easy hosting of your Videofront's Videoteca player via iframe embedding. 
+
+The deployment of this template creates a GitHub repository containing your project, as well as an edge application with several Cache Settings and a preconfigured route to launch the Videoteca Player edge function, as well as a domain to facilitate your access and management through Azion Edge Platform.
+
+
+## Requirements
+
+- A [GitHub account](https://github.com/signup) to connect with Azion and create your new repository.
+    - Every push will be deployed automatically to this repository to keep your project updated.
+- The [Videoteca Player integration](/en/products/guides/videoteca-player/) installed and configured in your Azion account.
+- This template uses [Application Accelerator](/en/documentation/products/build/edge-application/application-accelerator/), [Edge Functions](/en/documentation/products/build/edge-application/edge-functions/), [Image Processor](/en/documentation/products/build/edge-application/image-processor/) and [Edge Cache](https://www.azion.com/en/documentation/products/build/edge-application/edge-cache/), and it could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information.
```

Comment:
```suggestion
- This template uses [Application Accelerator](/en/documentation/products/build/edge-application/application-accelerator/), [Edge Functions](/en/documentation/products/build/edge-application/edge-functions/), [Image Processor](/en/documentation/products/build/edge-application/image-processor/) and [Edge Cache](/en/documentation/products/build/edge-application/edge-cache/), and it could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information.
```